### PR TITLE
Update currencies.ts and add IRR symbol

### PR DESCRIPTION
**Pull request by fpichlou**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by ericmartinezr** [comment]
You missed this part https://github.com/angular/angular/blob/c25ecce3ac78499bc588ef650d7636722b2bab6e/packages/common/src/i18n/currencies.ts#L9

**Comment by fpichlou** [comment]
@ericmartinezr Thanks!

**Comment by fpichlou** [comment]
> @googlebot I signed it!

File: packages/common/src/i18n/currencies.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const CURRENCIES_EN:
       'ILS': ['₪'],
       'INR': ['₹'],
       'IQD': [undefined, undefined, 0],
-      'IRR': [undefined, undefined, 0],
+      'IRR': [undefined, '﷼', 0 ],
```
Comment (mhevery) [inline packages/common/src/i18n/currencies.ts]:
```suggestion
      'IRR': [undefined, ﷼', 0' ],
```

**Review by mhevery — CHANGES_REQUESTED** [review]

**Comment by AndrewKushnir** [comment]
Hi @fpichlou, as @ericmartinezr mentioned, this file is auto-generated, so we can not merge this change. If there is a problem with the source CLDR then please raise an issue there, so that it can be resolved and we'll update it in Angular after that. Thank you.

**Pull request closed without merging**